### DISCUSSION TITLE
Welcome to Text-To-Video-Finetuning Discussions!

### DISCUSSION THREAD

**Discussion by ExponentialML** [ExponentialML/Text-To-Video-Finetuning#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cubey42** [comment]
{'variance_type'} was not found in config. Values will be initialized to default values.
Traceback (most recent call last):
  File "H:\AI video\Text-To-Video-Finetuning\train.py", line 488, in <module>
    main(**OmegaConf.load(args.config))
  File "H:\AI video\Text-To-Video-Finetuning\train.py", line 226, in main
    noise_scheduler, tokenizer, text_encoder, vae, unet = load_primary_models(pretrained_model_path)
  File "H:\AI video\Text-To-Video-Finetuning\train.py", line 75, in load_primary_models
    text_encoder = CLIPTextModel.from_pretrained(pretrained_model_path, subfolder="text_encoder")
  File "C:\Users\Lucas\AppData\Local\Programs\Python\Python310\lib\site-packages\transformers\modeling_utils.py", line 2274, in from_pretrained
    raise EnvironmentError(
OSError: Error no file named pytorch_model.bin, tf_model.h5, model.ckpt.index or flax_model.msgpack found in directory ./models/model_scope_diffusers/.

I only downloaded the fp16 models, do I need the full models for this to work?

**Reply by ExponentialML** [reply]
> {'variance_type'} was not found in config. Values will be initialized to default values. Traceback (most recent call last): File "H:\AI video\Text-To-Video-Finetuning\train.py", line 488, in main(**OmegaConf.load(args.config)) File "H:\AI video\Text-To-Video-Finetuning\train.py", line 226, in main noise_scheduler, tokenizer, text_encoder, vae, unet = load_primary_models(pretrained_model_path) File "H:\AI video\Text-To-Video-Finetuning\train.py", line 75, in load_primary_models text_encoder = CLIPTextModel.from_pretrained(pretrained_model_path, subfolder="text_encoder") File "C:\Users\Lucas\AppData\Local\Programs\Python\Python310\lib\site-packages\transformers\modeling_utils.py", line 2274, in from_pretrained raise EnvironmentError( OSError: Error no file named pytorch_model.bin, tf_model.h5, model.ckpt.index or flax_model.msgpack found in directory ./models/model_scope_diffusers/.
> 
> I only downloaded the fp16 models, do I need the full models for this to work?

Hey. Did you download the safetensors?
I need to update the requirements, but `pip install safetensors` should do the trick.

**Reply by Cubey42** [reply]
> > {'variance_type'} was not found in config. Values will be initialized to default values. Traceback (most recent call last): File "H:\AI video\Text-To-Video-Finetuning\train.py", line 488, in main(**OmegaConf.load(args.config)) File "H:\AI video\Text-To-Video-Finetuning\train.py", line 226, in main noise_scheduler, tokenizer, text_encoder, vae, unet = load_primary_models(pretrained_model_path) File "H:\AI video\Text-To-Video-Finetuning\train.py", line 75, in load_primary_models text_encoder = CLIPTextModel.from_pretrained(pretrained_model_path, subfolder="text_encoder") File "C:\Users\Lucas\AppData\Local\Programs\Python\Python310\lib\site-packages\transformers\modeling_utils.py", line 2274, in from_pretrained raise EnvironmentError( OSError: Error no file named pytorch_model.bin, tf_model.h5, model.ckpt.index or flax_model.msgpack found in directory ./models/model_scope_diffusers/.
> > I only downloaded the fp16 models, do I need the full models for this to work?
> 
> Hey. Did you download the safetensors? I need to update the requirements, but `pip install safetensors` should do the trick.

yes, I only downloaded the fp16 safetensors, I'm downloading the non-fp16 ones now.

**Reply by Cubey42** [reply]
downloading the other models worked, but now I have an even more confusing error;
 ValueError: Cannot load <class 'diffusers.models.autoencoder_kl.AutoencoderKL'> from ./models/model_scope_diffusers/ because the following keys are missing:
blahblahblah
 Please make sure to pass `low_cpu_mem_usage=False` and `device_map=None` if you want to randomly initialize those weights or else make sure your checkpoint file is correct.

**Reply by Cubey42** [reply]
okay it was just an extension of my previous issue, once I copied the whole directory, it worked. also wow even just from my test im getting some scary good results. insane what this is capable of

**Reply by ExponentialML** [reply]
@Cubey42 Glad to hear you got it working!

**Reply by sergiobr** [reply]
I got this on starting also. Now it's working fine.
Is there a way to git clone huggingface repo including all models?

**Reply by ExponentialML** [reply]
@sergiobr 

This should do the trick. I'll add it to the readme.
```
git lfs install
git clone https://huggingface.co/damo-vilab/text-to-video-ms-1.7b
```

**Reply by sergiobr** [reply]
Great I knew there were a command for it. Thank you.
